guvnor-structure/guvnor-structure-client/src/test/java/org/guvnor/structure/client/editors/repository/clone/mock/MockCloneRepositoryPresenter.java

```diff
@@ -0,0 +1,21 @@
+package org.guvnor.structure.client.editors.repository.clone.mock;
+
+import org.guvnor.structure.client.editors.repository.clone.CloneRepositoryPresenter;
+
+/**
+ * Created by jomarko on 7/15/15.
+ */
```

Comment:
We try to keep the code free of author tags for few reason that are explained here. 
https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/blob/master/README.md#faq
But I don't think the PR should be rejected because of this. This just gets removed once somebody spots it.